EIPS/eip-725.md

```diff
@@ -1,55 +1,93 @@
----
+```
```

Comment:
```suggestion
---
```
`---` was correct, this is not a code block, it is markdown frontmatter (YML encoded internally).